### PR TITLE
change symbol and logo for aseed token

### PR DESCRIPTION
**Pull request by topdevgh**

### CODE REVIEW

**Review by wolfwarrior14 — APPROVED** [review]
LGTM. Great Work!

File: src/constants/assets.ts [hunk 1]
```diff
@@ -111,9 +111,9 @@ export const ASSETS_DICT: { [key in AssetSymbol]: Asset } = {
   },
   aSEED: {
     symbol: 'aSEED',
-    displaySymbol: 'aUSD',
-    name: 'Acalla Dollar',
-    icon: SymbolAusd,
+    displaySymbol: 'aSEED',
+    name: 'Acala Dollar',
```
Comment (qudacki) [inline src/constants/assets.ts:115]: Please change to the new name `aUSD SEED`,
https://wiki.acala.network/acala-exodus-upgrade/ausd-seed-aseed#ausd-conversion-date

**Review by qudacki — CHANGES_REQUESTED** [review]

File: src/constants/assets.ts [hunk 2]
```diff
@@ -111,9 +111,9 @@ export const ASSETS_DICT: { [key in AssetSymbol]: Asset } = {
   },
   aSEED: {
     symbol: 'aSEED',
-    displaySymbol: 'aUSD',
-    name: 'Acalla Dollar',
-    icon: SymbolAusd,
+    displaySymbol: 'aSEED',
```
Comment (qudacki) [inline src/constants/assets.ts:114]: you can simply delete this field if same as the symbol

**Review by qudacki — APPROVED** [review]
good job!

**Review by wolfwarrior14 — APPROVED** [review]

**Pull request merged**